docs/usage.md

```diff
@@ -126,7 +126,9 @@ First, go to the [nf-core/mhcquant releases page](https://github.com/nf-core/mhc
 
 This version number will be logged in reports when you run the pipeline, so that you'll know what you used when you look back in the future.
 
-## Main arguments
+## Core Nextflow arguments
```

Comment:
The `## Core Nextflow arguments` section should go after the `input` and `fasta` sections. Additionally, as all parameters are now parsed from the parameters json file, you could remove these sections. If you want to leave them here, you could at least rename it to something like `Input files` and then link to further information on these parameters to the nf-core website